src/stories/Header.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
I don't think we want Header in here, do we? Was this a test?

Reply:
These are samples for the references and it won't be published as npm packages

Reply:
Fair enough. I see a lot of sample code in here we may be able to copy for our actual components.